bgpd/bgp_route.c

```diff
@@ -11081,7 +11102,8 @@ static int bgp_table_stats(struct vty *vty, struct bgp *bgp, afi_t afi,
 	thread_execute(bm->master, bgp_table_stats_walker, &ts, 0);
 
 	for (i = 0; i < BGP_STATS_MAX; i++) {
-		if (!table_stats_strs[i])
+		if ((!json && !table_stats_strs[i][0]) ||
```

Comment:
I think it's an interesting idea to add a table with one having the vty output and one having the json key.  We do need to provide some context around what 0 or 1 means, perhaps a define?

Reply:
ok